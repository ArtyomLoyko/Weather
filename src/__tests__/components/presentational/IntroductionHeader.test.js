import React from 'react';
import { shallow } from 'enzyme';
import IntroductionHeader from './../../../components/presentational/IntroductionHeader';

describe('IntroductionHeader', () => {
    const introductionHeader = shallow(<IntroductionHeader />);

    it('renders properly', () => {
        expect(introductionHeader).toMatchSnapshot();
    });
});
