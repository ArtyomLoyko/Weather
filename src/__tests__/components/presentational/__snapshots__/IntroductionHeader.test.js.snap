// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroductionHeader renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IntroductionHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          HELLO!!!
</h1>,
        <p>
          Find all pairs of cards as quickly as possible to break the record!
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "HELLO!!!",
        },
        "ref": null,
        "rendered": "HELLO!!!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Find all pairs of cards as quickly as possible to break the record!",
        },
        "ref": null,
        "rendered": "Find all pairs of cards as quickly as possible to break the record!",
        "type": "p",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            HELLO!!!
</h1>,
          <p>
            Find all pairs of cards as quickly as possible to break the record!
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "HELLO!!!",
          },
          "ref": null,
          "rendered": "HELLO!!!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Find all pairs of cards as quickly as possible to break the record!",
          },
          "ref": null,
          "rendered": "Find all pairs of cards as quickly as possible to break the record!",
          "type": "p",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
